helm/cluster-cloud-director/templates/cilium-helmrelease.yaml

```diff
@@ -29,7 +29,7 @@ spec:
   interval: 10m
   install:
     remediation:
-      retries: 3
+      retries: 30
```

Comment:
What is the remediation action here?

Reply:
Please see the PR description `kubectl explain helmrelease.spec.install.remediation.retries` output. My understanding is that it is the number retries that happen when `helm install` fails (but not `helm upgrade`).

Reply:
🤦 I missed that part. Sorry.